src/navigation/screens/Profile.tsx

```
@@ -1,21 +1,144 @@
+import { useNavigation } from '@react-navigation/native';
+import { selectUser } from '../../store/slices/authSlice';
+import { colors, typography, spacing, borderRadius, darkColors, lightColors } from '../../utils/theme';
+import { styles } from './styles/ProfileStyles';
+import { selectDarkMode } from '../../store/slices/themeSlice';
+import { setDarkMode } from '../../store/slices/themeSlice';
 
 export function Profile() {
+  const navigation = useNavigation();
+ 
+  // get user data from redux
+  const user = useSelector(selectUser);
```

ESLint

[error] 'user' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)